tsconfig.json

```diff
@@ -9,11 +9,17 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "esModuleInterop": true,
-    "lib": ["ESNext", "DOM"],
+    "lib": [
+      "ESNext",
+      "DOM"
+    ],
     "skipLibCheck": true,
     "noEmit": true,
+    "allowImportingTsExtensions": true,
```

Comment:
I don't think we need this. When importing `./foo.ts` in Node, we can just use `./foo`.